packages/client/src/workflow-client.ts

```diff
@@ -909,6 +950,25 @@ export class WorkflowClient {
           headers: {},
         }) as Promise<Ret>;
       },
+      async fetchHistory(): Promise<FetchedHistory> {
+        // TODO: Interceptors? Bleh
```

Comment:
Yeah, it's a tough call. We originally only did it on things that might want interceptors, but then I was told that was wrong the other day when we added interceptors for "workflow describe". So now basically the rule seems to be "interceptors for all high-level client calls".

Reply:
Sounds like we don't need to if there are no headers according to @bergundy and I'll take that answer and run with it